distribution/docs/src/main/resources/_contents/_geowebcache-contents/managing-geowebcache-contents.adoc

```diff
@@ -0,0 +1,127 @@
+
+=== Prerequisites for ${ddf-geowebcache}
+
+none.
+
+=== Installing ${ddf-geowebcache}
+
+The ${ddf-geowebcache} application is *not* installed by default. To install:
+
+* Navigate to the ${admin-console}.
+* Select *Manage*.
+* Select the application *Start* (icon:play[]) button. The application will move to *Active Applications* on startup.
+* Select *Back* in ${admin-console}.
+
+=== Configuring ${ddf-geowebcache}
+
+==== Adding GeoWebCache Layers
+
+* Navigate to the ${admin-console}.
+* Select the ${ddf-geowebcache} Application.
+* Select the *GeoWebCache Layers* tab.
+* Click the *Add* (icon:plus[]) button.
+* Enter the data in the fields provided.
+* If necessary, click the *Add* button (icon:plus[]) to add additional MIME types.
+* If necessary, click the *Add* button (icon:plus[]) to add additional WMS Layer Names.
+
+.Add Layer
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Name
+|
+|String
+|
+|
+|no
+
+|Mime Formats
+|
+|String
+|
+|
+|yes
+
+|URL
+|
+|
+|
+|
+|yes
+
+|WMS Layer Name
+|
+|String
+|
+|
+|yes
+
+|===
+
+==== Editing GeoWebCache Layers
+
+* Navigate to the ${admin-console}.
+* Select the ${ddf-geowebcache} application.
+* Navigate to the *GeoWebCache Layers* tab.
+* Click the *Name* field of the layer to edit.
+
+==== Removing GeoWebCache Layers
+
+* Click the *Delete* icon (icon:trash[]) at the end of the row of the layer to be deleted.
+
+==== Configuring GWC Disk Quota
+
+Storage usage for a GeoWebCache server is managed by a `diskquota.xml` file with configuration details to prevent image-intensive data from filling the available storage.
+
+To view the disk quota XML representative: \https://localhost:8993/geowebcache/rest/diskquota.xml"
+
+To update the disk quota, a client can post a new XML configuration: `curl -v -k -XPUT -H "Content-type: text/xml" -d @diskquota.xml "https://localhost:8993/geowebcache/rest/diskquota.xml"`
+
+.Example `diskquota.xml`
+[source,xml,linenums]
+----
+<gwcQuotaConfiguration>
+  <enabled>true</enabled>
+  <diskBlockSize>2048</diskBlockSize>
+  <cacheCleanUpFrequency>5</cacheCleanUpFrequency>
+  <cacheCleanUpUnits>SECONDS</cacheCleanUpUnits>
+  <maxConcurrentCleanUps>5</maxConcurrentCleanUps>
+  <globalExpirationPolicyName>LFU</globalExpirationPolicyName>
+  <globalQuota>
+    <value>100</value>
+    <units>GiB</units>
+  </globalQuota>
+  <layerQuotas/>
+</gwcQuotaConfiguration>
+----
+
+See http://geowebcache.org/docs/current/configuration/diskquotas.html[Disk Quotas] for more information on configuration options for disk quota.
+
+==== Configuring the Standard Search UI for GeoWebCache
+
+Add a new Imagery Provider in the ${admin-console}:
+
+* Navigate to \https://<localhost>:8993/admin.
+* Select *Configuration* tab.
+* Select *Standard Search UI* configuration.
+* Click the *Add* button (icon:plus[]) next to *Imagery Providers*
```

Comment:
plan b was to change them to images, but the pdf engine is flaky for them too, so I just dropped them altogether.